deps/Versions.make

```diff
@@ -1,6 +1,6 @@
 LLVM_VER = 3.3
 LLVM_LIB_SUFFIX =
-PCRE_VER = 10.20
+PCRE_VER = 10.10
```

Comment:
Sorry -- just accidentally pushed something I need in my local build (https://github.com/JuliaLang/julia/issues/12038#issuecomment-143021496). Will be edited out.